### DISCUSSION TITLE
tshock not opening, opens but immediately crashes, any help?

### DISCUSSION THREAD

**Discussion by ratticuse** [Pryaxis/TShock#2916] [Q&A]
so I have been trying to make a server on t-shock, but I cant get the easiest thing done, so I try to open t-shock opens but immediately crashes I have no clue as of what is going on so can someone give me info to try to resolve the situation, thx. and there is no error code or nothing.
![2023-02-14 17-04-03](https://user-images.githubusercontent.com/125344264/218663628-c4a88624-3e37-432d-b08d-eceaffeada8c.gif)

**Comment by sgkoishi** [comment]
See https://github.com/Pryaxis/TShock/discussions/2843 (scroll down for comments)
